content/docs/1.0.0/advanced-resources/rwx-workloads.md

```diff
@@ -1,72 +1,74 @@
 ---
-title: RWX workloads
+title: Support for ReadWriteMany (RWX) workloads
 weight: 4
 ---
 
-# Longhorn RWX support
+Longhorn supports RWX workloads via the **nfs.longhorn.io** provisioner.
+
+The following diagram shows the requirements for these workloads:
```

Comment:
The diagram shows how the `rwx support` works. Not sure about `requirements`

Reply:
updated